docs/SUMMARY.md

```diff
@@ -19,7 +19,6 @@
     -   [@trezor/connect](./packages/connect/index.md)
         -   [debugging](./packages/connect/debugging.md)
         -   [dependencies](./packages/connect/dependencies.md)
-        -   [deployment](./packages/connect/deployment.md)
```

Comment:
You are removing this because now this information is included in `packages/connect/README.md`, right?

Reply:
yes. this file does not even exist anymore

Reply:
I see `deployment.md` file was already remove from `develop`